int32 bumper1
int32 bumper2
int32 bumper3
int32 bumper4
int32 bumper5
int32 bumper6
int32 bumper7
int32 bumper8
